projects/botfalcon

```diff
@@ -0,0 +1,37 @@
+import { FetchOptions, SimpleAdapter } from "../adapters/types";
+import { CHAIN } from "../helpers/chains";
+import { queryDune } from "../helpers/dune";
+
+const fethcFeesSolana = async (options: FetchOptions) => {
+  try {
+    const dailyFees = options.createBalances();
+    const value = (await queryDune("3521814", {
+      start: options.startTimestamp,
+      end: options.endTimestamp,
+      receiver: 'EaJZ8BsVJtxjQmzhaFJV5igpcbYoF9eWSycGhFqwhXU6'
```

Comment:
what is this address? and why this file in fee adapter format instead of tvl adapter format, does the protocol have tvl?

Reply:
protocol does not have TVL, its a Telegram Bot.
This address are the fee address.